Formula/xrootd.rb

```diff
@@ -23,8 +23,13 @@ class Xrootd < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "libcython" => :build
+  depends_on "python@3.10" => [:build, :test]
+  depends_on "davix"
+  depends_on "krb5"
   depends_on "openssl@1.1"
   depends_on "readline"
+  depends_on "tinyxml"
```

Comment:
No, this is deprecated upstream. It should not be added as a dependency to anything

Reply:
Ok, this is not a required dependency, I can remove. However, while the package is there, it would be nice to keep it if possible. I will try to move upstream (which I am part of) to `tinyxml2` at least so in a future version this dependency will be gone.

Reply:
We're trying to deprecate tinyxml since upstream has deprecated it a long time ago. Adding it for optional dependencies on a package would prevent that so it's not acceptable.

Reply:
Removed.